examples/chain-extension/Cargo.toml

```
@@ -1,33 +1,35 @@
 [package]
 name = "chain-extension"
-authors = ["Cardinal", "Aleph Zero Foundation"]
+authors = ["Use Ink <ink@use.ink>"]
```

```suggestion
authors = ["Cardinal", "Aleph Zero Foundation", "Use Ink <ink@use.ink>"]
```
Ditto about authors.